README.md

```diff
@@ -343,24 +343,46 @@ u.addresses.where(:city => 'Chicago').all
 
 But keep in mind Dynamoid -- and document-based storage systems in general -- are not drop-in replacements for existing relational databases. The above query does not efficiently perform a conditional join, but instead finds all the user's addresses and naively filters them in Ruby. For large associations this is a performance hit compared to relational database engines.
 
-You can also limit the number of evaluated records, or select a record from which to start, to support pagination:
+#### Limits
+
+There are three types of limits that you can query with:
+
+1. `record_limit` - The number of evaluated records that are returned by the query.
+2. `scan_limit` - The number of scanned records that DynamoDB will look at before returning.
+3. `batch_size` - The number of records requested to DynamoDB per underlying request, good for large queries!
```

Comment:
Perhaps we should mention that there is a maximum batch size set my Amazon, currently documented as 100.
```
BatchGetItem
A single BatchGetItem operation can retrieve a maximum of 100 items. The total size of all the items retrieved cannot exceed 16 MB.
```
http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html